### DISCUSSION TITLE
Ontology/Vocabulary for talking about the FAIRness of data?

### DISCUSSION THREAD

**Discussion by clange** [IndustrialDataSpace/InformationModel#507] [Q&A]
I am currently thinking about how to apply the IDS Infomodel for talking about how AI-ready data are. In our conceptual model, the Concern Hexagon, that's an aspect of Commodity/Quality. Indeed various quality dimensions/metrics for AI-readiness could be defined using the [Data Quality Vocabulary](https://www.w3.org/TR/vocab-dqv), which we recommend for talking about quality. However, given that FAIR has not only been interpreted as "Findable, Accessible, Interoperable, Reusable", but also as "fully AI ready" and given that the [FAIR Principles](https://www.go-fair.org/fair-principles/) are well understood, I wonder whether there exists a ready to reuse ontology to talk about the FAIRness of data. It's hard to search the Web for this, because you find many results for the related concept of FAIRness of ontologies.

**Comment by aliariff** [comment]
maybe this?
- https://peta-pico.github.io/FAIR-nanopubs/principles/index-en.html
- https://github.com/FAIRMetrics/Metrics/
